admission_control/admission_control_proto/src/proto/mod.rs

```diff
@@ -6,7 +6,6 @@
 use ::types::proto::*;
 use mempool::proto::shared::mempool_status;
 
-/// Auto generated proto src files
-pub mod admission_control;
-/// Auto generated proto src files
-pub mod admission_control_grpc;
+pub mod admission_control {
+    include!(concat!(env!("OUT_DIR"), "/admission_control.rs"));
```

Comment:
left you this comment elsewhere, but I think i'd be better to be able to keep the `pub mod admission_control` and have the generated files here instead of `target/` (if the OUT_DIR is pointing to target?)

but now I'm reading this:

> In general, build scripts should not modify any files outside of OUT_DIR. It may seem fine on the first blush, but it does cause problems when you use such crate as a dependency, because there's an implicit invariant that sources in .cargo/registry should be immutable. cargo won't allow such scripts when packaging.

from https://doc.rust-lang.org/cargo/reference/build-scripts.html

but the idea of having actual files that will be used in target/ feels weird to me

Reply:
Yeah that section basically states that you shouldn't have a build script modify your src dir and the method prost uses is the preferred mechanism for including generated code.